tasks/e2e-kitchensink.sh

```diff
@@ -143,12 +145,19 @@ PORT=3001 \
   REACT_APP_SHELL_ENV_MESSAGE=fromtheshell \
   NODE_PATH=src \
   nohup npm start &>$tmp_server_log &
-grep -q 'The app is running at:' <(tail -f $tmp_server_log)
+while true
+do
+  if grep -q 'The app is running at:' $tmp_server_log; then
+    break
+  else
+    sleep 1
+  fi
+done
 E2E_URL="http://localhost:3001" \
   REACT_APP_SHELL_ENV_MESSAGE=fromtheshell \
   CI=true NODE_PATH=src \
   NODE_ENV=development \
-  node node_modules/.bin/mocha --require babel-register --require babel-polyfill integration/*.test.js
```

Comment:
Yeah, it's useless ;)
I'm on a mobile right now, so no code on hand, but this useless `node` might appear somewhere else